src/Imghoard.Tests/ClientTests.cs

```diff
@@ -10,9 +10,30 @@ public class ClientTests
         [Fact]
         public void ConstructClientTest()
         {
-            var i = new ImghoardClient(ImghoardClient.Config.Default());
+            var i = new ImghoardClient();
 
             Assert.NotNull(i);
+            Assert.Equal(Config.Default().Endpoint, i.GetEndpoint());
+        }
+
+        [Fact]
+        public async Task GetSingleImageAsync()
+        {
+            var mock = new Mock<IImghoardClient>();
+
+            mock.Setup(x => x.GetImageAsync(It.IsAny<ulong>()))
```

Comment:
Technically here you want to mock the http client and inject it into the imghoard client